### ISSUE TITLE
Documentation links not working

### ISSUE THREAD

**Issue by gdubicki** [tsteur/open-source-tag-manager#2]
"Sorry.

This URL has been excluded from the Wayback Machine."

Do you have a copy of the original docs which you can share?

**Comment by gdubicki** [comment]
To be specific I encountered this issue when trying to follow https://github.com/tsteur/open-source-tag-manager/blob/master/doc/INSTALLATION.md : https://github.com/liip/LiipMonitorBundle/issues/147 . Do you know anything about it?

**Comment by gregorvolkmann** [comment]
Hey @gdubicki, could you install it another way?

**Comment by gdubicki** [comment]
No, I stopped trying assuming that it makes no sense if it does not "just work" when it is unsupported.
